docs/asciidoc/guides/2.0.0-migration/index.adoc

```diff
@@ -0,0 +1,18 @@
+
+[[rel-migration]]
+= 2.0.0 Migration
+
+Version 2.0.0 of `@neo4j/graphql` adds support for relationship properties, with some breaking changes to facilitate these new features. All of the required changes will be on the client side, and this guide will walk through what has changed.
+
+== How to Upgrade
+
+Simply update `@neo4j/graphql` using npm or your package manager of choice:
+
+[source, bash]
+----
+npm update @neo4j/graphql
```

Comment:
Will this work due to the fact it will be alpha and npm won't push users onto an unstable version?

Reply:
It won't, but I've written it in the tense of "we've released 2.0.0 stable" so that it's ready to merge into master whenever we are. Do you think that will be fine?